scripts/get_weather.py

```diff
@@ -0,0 +1,24 @@
+import requests, json
+
+BASE = "https://api.openweathermap.org/data/2.5/weather?"
+APIKEY = "fc2822e01a0efcce52cb56212fe44fd5"
+CITY = "Stony Brook"
+URL = BASE + "q="+ CITY + "&appid=" + APIKEY
+response = requests.get(URL)
+
+if response.status_code == 200:
+    data = response.json()
+    main = data['main']
+    temperature = main['temp'] #in Kelvin
+    tempF = (temperature - 273.15) * (9/5) + 32
+    humidity = main['humidity']
+    pressure = main['pressure']
```

Comment:
Let's clean up the code where you're accessing from `main` and `data`
```
main, weather = data['main'], data['weather']
humidity, pressure = main['humidity'], main['pressure']
```

Take advantage of python's syntax tricks to cut down the line of code. If you're not leveraging these feature there's no point in using python. In general, it is better to have less lines of code in order to reduce the probability of introducing a bug even if the code executing in this script is innocuous.